src/components/button/BasicButton.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Button, ButtonProps, styled } from "@material-ui/core";
+import { COLORS } from "../../utils/colors";
+
+const StyledButton = styled(Button)({
```

Comment:
Is there any reason why we are using `styled` in some places and `makeStyles` in others?